data_viz/send_data.py

```diff
@@ -0,0 +1,19 @@
+'''
+    A quick script to send data as post requests
+'''
+
+import requests 
+
+url = "http://localhost:8000/data_viz/api/nodes/"
+data1 = "CellC - 680mA DischCurr.xlsx"
```

Comment:
What is this for? WIll we be using this document?

Reply:
Yes, for the alumni event. Laila sent me this data and I figured we could run a 'pseudo-realtime' trial with some real data.